src/screens/SpeakersScreen.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { View, FlatList, StyleSheet, Text} from 'react-native';
+import SpeakerCard from '../components/cards/SpeakerCard';
+import type { SpeakerCardProps } from '../components/cards/SpeakerCard';
+import BackArrowIcon from '../assets/icons/BackArrowIcon';
+import { colors } from "../constants/Colors";
+import { fonts } from '../assets/fonts/fonts';
+
+  const SpeakersScreen = (props : {SpeakersData : SpeakerCardProps[] }) => {
```

Comment:
Good work on the Speaker's screen!

I have noticed you defined the screen as a component without any navigation props like the [LogIn Screen Component](https://github.com/droidconKE/droidconKE2022ReactNative/blob/develop/src/screens/LoginScreen.tsx) ( see lines 11-19 )

In order to access a screen, it needs navigation props passed to it & also declared on MainStackNavigator. 
However, maybe it is possible to access Speaker's screen without navigation props. What do you think about this?

Reply:
@brianwachira I see what you meant earlier. I guess this also works. Since we are currently not using navigation prop, we can always add it when we need it. I think the screen will just render.

Reply:
> @brianwachira I see what you meant earlier. I guess this also works. Since we are currently not using navigation prop, we can always add it when we need it. I think the screen will just render.

Okay, I just thought it would be better if the screen was plug and play such that everything necessary has already been set, including navigation props.😅​

Reply:
Let me try adding the navigation prop : )

Reply:
I could use redux for the Speakers data and use the navigation props on the screen @brianwachira @makunomark

Reply:
That is cool!
If you can implement that then go ahead!